pkg/db/pgx_interface.go

```diff
@@ -3,50 +3,31 @@ package db
 import (
 	"context"
 
-	"github.com/jackc/pgx"
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgconn"
 )
 
 // An implicit interface representing what pgx.ConnPool is offering.
 // Only the needed methods are exposed.
 type pgxConnectionPool interface {
 	Close()
 
-	BeginEx(ctx context.Context, txOptions *pgx.TxOptions) (*pgx.Tx, error)
+	Begin(ctx context.Context) (pgx.Tx, error)
 
-	QueryEx(ctx context.Context, sql string, options *pgx.QueryExOptions, arguments ...interface{}) (*pgx.Rows, error)
-	ExecEx(ctx context.Context, sql string, options *pgx.QueryExOptions, arguments ...interface{}) (pgx.CommandTag, error)
-}
-
-// An implicit interface representing what pgx.Tx is offering.
-// Only the needed methods are exposed.
-type pgxTransaction interface {
-	RollbackEx(ctx context.Context) error
-	CommitEx(ctx context.Context) error
-
-	QueryEx(ctx context.Context, sql string, options *pgx.QueryExOptions, arguments ...interface{}) (*pgx.Rows, error)
-	ExecEx(ctx context.Context, sql string, options *pgx.QueryExOptions, arguments ...interface{}) (commandTag pgx.CommandTag, err error)
-
-	Err() error
-}
```

Comment:
Those are now defined as interfaces (see [Tx](https://github.com/jackc/pgx/blob/60a01d044a5b3f65b9eea866954fdeea1e7d3f00/tx.go#L118) for the transaction): therefore we don't need our own custom interfaces anymore :+1: